packages/appkit/src/client.ts

```diff
@@ -446,8 +446,10 @@ export class AppKit {
     }
 
     this.adapters = options.adapters
-
-    this.setMetadata(options)
+    
+    if (!options.metadata) {
```

Comment:
Wouldn't it be better to have a default metadata value and always set the metadata? 

Also, if there IS a metadata, I don't see where it's being set